[2024-08-16T13:59:27.909+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: update_data_after_retrain.push_newDataTrain_from_temp manual__2024-08-16T13:59:18.714930+00:00 [queued]>
[2024-08-16T13:59:27.931+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: update_data_after_retrain.push_newDataTrain_from_temp manual__2024-08-16T13:59:18.714930+00:00 [queued]>
[2024-08-16T13:59:27.932+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-08-16T13:59:28.014+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): push_newDataTrain_from_temp> on 2024-08-16 13:59:18.714930+00:00
[2024-08-16T13:59:28.024+0000] {standard_task_runner.py:60} INFO - Started process 13308 to run task
[2024-08-16T13:59:28.033+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'update_data_after_retrain', 'push_newDataTrain_from_temp', 'manual__2024-08-16T13:59:18.714930+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/update_data_after_retrain.py', '--cfg-path', '/tmp/tmpwvbhi1pj']
[2024-08-16T13:59:28.039+0000] {standard_task_runner.py:88} INFO - Job 20: Subtask push_newDataTrain_from_temp
[2024-08-16T13:59:28.084+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-08-16T13:59:28.166+0000] {task_command.py:423} INFO - Running <TaskInstance: update_data_after_retrain.push_newDataTrain_from_temp manual__2024-08-16T13:59:18.714930+00:00 [running]> on host 282fa7a8e0f3
[2024-08-16T13:59:28.440+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='update_data_after_retrain' AIRFLOW_CTX_TASK_ID='push_newDataTrain_from_temp' AIRFLOW_CTX_EXECUTION_DATE='2024-08-16T13:59:18.714930+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-16T13:59:18.714930+00:00'
[2024-08-16T13:59:28.518+0000] {logging_mixin.py:188} INFO - retrain: False
[2024-08-16T13:59:28.519+0000] {update_data_after_retrain.py:36} INFO - retrain value from task 3 : False
[2024-08-16T13:59:28.522+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-08-16T13:59:28.591+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=update_data_after_retrain, task_id=push_newDataTrain_from_temp, execution_date=20240816T135918, start_date=20240816T135927, end_date=20240816T135928
[2024-08-16T13:59:28.699+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-08-16T13:59:28.765+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
